react-native/react/common-adapters/checkbox.js.flow

```diff
@@ -5,8 +5,8 @@ import React, {Component} from '../base-react'
 export type Props = {
   label: string,
   onCheck: Function,
-  checked: bool,
-  style: object
+  checked: boolean,
```

Comment:
We should make a ticket to unify all those in our code

On Tuesday, January 12, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/react/common-adapters/checkbox.js.flow
> <https://github.com/keybase/client/pull/1701#discussion_r49491792>:
>
> > @@ -5,8 +5,8 @@ import React, {Component} from '../base-react'
> >  export type Props = {
> >    label: string,
> >    onCheck: Function,
> > -  checked: bool,
> > -  style: object
> > +  checked: boolean,
>
> it should be boolean according to the docs (
> http://flowtype.org/docs/quick-reference.html#built-in-primitive-types)
> but they use both in the flow code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1701/files#r49491792>.
>